tcms/testruns/tests/test_views.py

```diff
@@ -824,3 +825,23 @@ def test_should_fail_when_try_to_change_status_without_permissions(self):
         self.assertRedirects(
             self.client.get(self.url, {'finished': 1}),
             reverse('tcms-login') + '?next=%s?finished=1' % self.url)
+
+
+class TestCreateTestUsers(AssignPermissionsToTestUser, TestChangeTestRunStatus):
```

Comment:
Multiple inheritance is not a good idea here, especially b/c both of the classes have common ancestors and common methods.